packages/server/lib/open_project.coffee

```diff
@@ -209,10 +212,11 @@ create = ->
       ## open the project and return
       ## the config for the project instance
       debug("opening project %s", path)
+      debug("and options %o", options)
```

Comment:
```js
debug('opening project %o', { path, options }')
```

Reply:
so I split it across two lines because options is a large object, which kind of hides the path. By having these two on separate lines I get nice stdout that's easy to scan